promql/promqltest/testdata/native_histograms.test

```diff
@@ -269,3 +269,18 @@ eval instant at 50m histogram_sum(sum(incr_sum_histogram))
 
 eval instant at 50m histogram_sum(sum(last_over_time(incr_sum_histogram[5m])))
     {} 30
+
+
+
+# Test native histograms with custom buckets.
+load 5m
+	custom_buckets_histogram	{{schema:-53 sum:5 count:4 custom_values:[5 10] buckets:[1 2 1]}}x10
```

Comment:
I agree it'd be good to make this all consistent. Since this is not directly related to NHCB though, can we do this in a separate PR with only the whitespace changes after this is merged? Currently the files are all inconsistent so this would change many lines and would make it look like this PR affects more tests than it actually does (making it hard to see the actual changes), and would cause more merge conflicts if people modify tests elsewhere. For now, I made sure the tests I added have tabs/spaces consistent with similar/adjacent tests in the same file.

Reply:
I would even say it's fine if every block is self-consistent. If it changes throughout the file, it's not that bad. Just having lines next to each other with different alignment looks very weird.